src/Core/Managed/Net45/Extensibility/Implementation/RichPayloadEventSource.cs

```diff
@@ -78,6 +83,40 @@ public void Process(ITelemetry item)
         }
 
         /// <summary>
+        /// Record an operation start.
+        /// </summary>
+        /// <param name="operation">The opeation which is about to start.</param>
+        public void ProcessOperationStart(OperationTelemetry operation)
+        {
+            if (this.EventSourceInternal == null)
+            {
+                return;
+            }
+
+            if (this.EventSourceInternal.IsEnabled(EventLevel.Informational, Keywords.Operations))
+            {
+                operationStartStopHandler(operation, EventOpcode.Start);
+            }
+        }
+
+        /// <summary>
+        /// Record an operation stop.
+        /// </summary>
+        /// <param name="operation">The operation which has just stopped.</param>
+        public void ProcessOperationStop(OperationTelemetry operation)
```

Comment:
My point was that for every request we will get `Start`, then `Stop` and then once again `Track`. Three events instead of 2. Do we need to change `TrackRequest` to report `Stop` instead?